.github/workflows/json.yml

```diff
@@ -0,0 +1,66 @@
+name: Json
+
+on: [push, pull_request]
+
+jobs:
+  pass:
+    name: >-
+      pass ${{ matrix.os }} ${{ matrix.ruby }}
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu, macos, windows ]
+        ruby: [ head, 2.7, 2.6, 2.5, 2.4, 2.3, 2.2, truffleruby ]
+        include:
+          - { os: windows, ruby: mingw }
+          - { os: windows, ruby: mswin }
+        exclude:
+          - { os: windows, ruby: head }
+          - { os: windows, ruby: truffleruby }
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: MSP-Greg/setup-ruby-pkgs@v1
```

Comment:
Can you remove the your actions? The maintainers can't control it.

Reply:
It's needed to install `ragel` in a platform-agnostic way.
`MSP-Greg/setup-ruby-pkgs@v1` just uses `ruby/setup-ruby` for the setting up Ruby part.